.ci_support/linux_64_.yaml

```diff
@@ -1,8 +1,16 @@
 cdt_name:
-- cos7
+- conda
 channel_sources:
 - conda-forge
 channel_targets:
 - conda-forge diffpy.fourigui_rc
 docker_image:
 - quay.io/condaforge/linux-anvil-cos7-x86_64
+pin_run_as_build:
+  python:
+    min_pin: x.x
+    max_pin: x.x
+python:
+- 3.12.* *_cpython
+python_min:
+- '3.9'
```

Comment:
is there a reason this is 3.9?

Reply:
It's automatically generated but I checked our linux CI that `3.11` is used as we've instructed in `meta.yaml`

```
============================= test session starts ==============================
platform linux -- Python 3.11.10, pytest-8.3.3, pluggy-1.5.0
rootdir: $SRC_DIR
plugins: mock-3.14.0
```